tests/api_tests/nodes/test_views.py

```diff
@@ -1520,3 +1524,116 @@ def test_delete_deleted_public_node(self):
     def test_delete_deleted_private_node(self):
         res = self.app.delete(self.private_url, auth=self.user.auth, expect_errors=True)
         assert_equal(res.status_code, 410)
+
+class TestNodeLogList(ApiTestCase):
+    def setUp(self):
+        super(TestNodeLogList, self).setUp()
+        self.user = AuthUserFactory()
+        self.contrib = AuthUserFactory()
+        self.creator = AuthUserFactory()
+        self.user_auth = Auth(self.user)
+        self.NodeLogFactory = ProjectFactory()
+        self.pointer = ProjectFactory()
+        self.private_project = ProjectFactory(is_public=False, creator=self.user)
+        self.private_url = '/{}nodes/{}/logs/'.format(API_BASE, self.private_project._id)
+
+        self.public_project = ProjectFactory(is_public=True, creator=self.user)
+        self.public_url = '/{}nodes/{}/logs/'.format(API_BASE, self.public_project._id)
+
+    def tearDown(self):
+        super(TestNodeLogList, self).tearDown()
+        NodeLog.remove()
+
+    def test_add_tag(self):
+        user_auth = Auth(self.user)
+        res = self.app.get(self.public_url, auth=self.user.auth)
+        assert_equal(res.status_code, 200)
+        self.public_project.add_tag("Jeff Spies", auth=user_auth)
+        assert_equal("tag_added", self.public_project.logs[-1].action)
+        res = self.app.get(self.public_url, auth=self.user.auth)
+        assert_equal(res.json['data'][-1]['action'], 'tag_added')
+        assert_equal("Jeff Spies", self.public_project.logs[-1].params['tag'])
+
+    def test_remove_tag(self):
+        user_auth = Auth(self.user)
+        res = self.app.get(self.public_url, auth=self.user.auth)
```

Comment:
You don't need this line nor the next one. We have tests for getting nodes, so no need to repeat that in these tests unless you're asserting something specific for the test.